packages/orchestration/src/cosmos-api.ts

```diff
@@ -212,12 +212,18 @@ export interface IcaAccount {
     msgs: AnyJson[],
     opts?: Partial<Omit<TxBody, 'messages'>>,
   ) => Promise<string>;
-  /** get Purse for a brand to .withdraw() a Payment from the account */
-  getPurse: (brand: Brand) => Promise<Purse>;
   /**
-   * Close the remote account
+   * Closes the ICA channel (account). Does not retrieve assets, so warn the
+   * caller to retrieve them first. However, the Port is persisted and
+   * holders can always call .reopen() to re-establish a connection.
    */
   close: () => Promise<void>;
+  /**
+   * Reopens teh ICA channel (account) that was previously closed by the holder.
+   * If a channel is closed for an unexpected reason, it should automatically
+   * reopen and the holder should not need to call .reopen().
```

Comment:
```suggestion
   * Reopens the ICA channel (account) if it was previously closed by `close()`.
   * If a channel is closed for an expect reason, an automatic attempt to reopen
   * will be made and the holder should not need to call .reopen().
```
What are the expects reasons?
What happens if the channel is already open? Noop or rejection?

Reply:
This all seems to be about _connecting_ which makes me think the `close` term is off. Closing an account, I'd expect to get the assets but the account isn't closed; it's still open. We're just disconnecting from or re-connecting to it. Let's discuss.

Reply:
> What are the expects reasons?

I am not sure if we will get something meaningful through the "reason". My current read is that it's a `network.js` convention and we'll always get "close-reason-finalizer". None of the code relies on the reason for any of the `reactivate` logic.

WRT the channel closure itself, the jsdoc-linked [Cosmos SDK doc](https://docs.cosmos.network/v0.45/ibc/overview.html#:~:text=In%20ORDERED%20channels%2C%20a%20timeout%20of%20a%20single%20packet%20in%20the%20channel%20closes%20the%20channel), while sparse, seems to describe best:

 - _In ORDERED channels, a timeout of a single packet in the channel closes the channel._


> What happens if the channel is already open? Noop or rejection?

I've included logic in `IcaAccountKit` that throws an error on calls to `reactivate()` if there's an active connection (`connection` object in state). At the ICS-27 protocol level, I'm not sure. I'd expect the host to no longer honor the authority of the current channel/connection and rely on the new one. 


> This all seems to be about connecting which makes me think the close term is off. 

Renamed to `deactivate()` and `reactivate()` as per discussion. `close()` does not accurately describe the behavior since assets aren't returned, and the account can be reopened. `connect()` and `reconnect()` makes sense at the network level, but less so at the `ICAAccount` level.